README.md

```diff
@@ -46,13 +46,13 @@ curl -XPOST 'http://localhost:8086/write?db=mydb' \
 
 ### Query for the data
 ```JSON
-curl -G http://localhost:8086/query?pretty=true --data-urlencode "db=mydb" \
+curl -G 'http://localhost:8086/query?pretty=true' --data-urlencode "db=mydb" \
```

Comment:
Can you change this to double quotes? I would like to keep it consistent.

Reply:
Thanks for your review. Done!